msal/src/test/java/com/microsoft/identity/client/e2e/tests/AcquireTokenAbstractTest.java

```diff
@@ -82,39 +82,43 @@ public void performInteractiveAcquireTokenCall(final String username, final Stri
         flushScheduler();
     }
 
-    public void performSilentAcquireTokenCall(final String[] scopes) {
+    public void performSilentAcquireTokenCall(final String[] scopes,
+                                              final boolean expectingResultFromCache) {
         final AcquireTokenSilentParameters silentParameters = new AcquireTokenSilentParameters.Builder()
                 .forAccount(getAccount())
                 .fromAuthority(getAuthority())
                 .withScopes(Arrays.asList(scopes))
                 .forceRefresh(false)
-                .withCallback(successfulSilentCallback())
+                .withCallback(successfulSilentCallback(expectingResultFromCache))
                 .build();
 
         mApplication.acquireTokenSilentAsync(silentParameters);
         flushScheduler();
     }
 
-    public void performSilentAcquireTokenCall(final IAccount account) {
+    public void performSilentAcquireTokenCall(final IAccount account,
+                                              final boolean expectingResultFromCache) {
```

Comment:
Same here

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1048/commits/f09a7afc15134b05fd6c23615308bfa4ea53dbdf